src/panel/poi/blocks/helpers.js

```diff
@@ -0,0 +1,3 @@
+export function hasDisplayableDelivery(delivery) {
+  return !!delivery && Object.entries(delivery).some(([_key, value]) => value === 'yes');
```

Comment:
This is not very future-proof. 
If a new key is added in the API response (we would expect that to be backward-compatible), the `DeliveryBlock` may be displayed whereas none of the 3 known labels is enabled. 

So this logic is necessarily intertwined with the rendering of the block. Perhaps it would make sense to define such an helper in the same file as the block itself?

Reply:
Indeed! I changed it.
Thanks for the suggestion of bringing the helper and the block together, it may be the cleaner approach for all these blocks actually.